torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -41,6 +45,20 @@ std::shared_ptr<FutureMessage> py_rpc_builtin(
         Stack stack = torch::jit::createStackForSchema(
```

Comment:
This is not part of this PR, but I noticed it while reviewing the code here. Above, in line 38, there is a `opName.rfind("aten", 0) == 0`. That's a code smell to me: I generally don't think string matching in this way is a good idea, because it's easy to get confounded by cases like `"atenblah"`. Is there a reason we can't use `Symbol::fromQualString` and then test if the namespace is ATen? That seems semantically better to me.